templates/web.letsencrypt.ssl.template.yml

```diff
@@ -51,6 +64,12 @@ run:
         }
         EOF
 
+        if [ -f "/proc/net/if_inet6" ] ; then
```

Comment:
Does this file exist if IPv6 is enabled but no interfaces have v6 addresses?

We want to listen on `::` even if there are current addresses, in case one appears in the future.

Reply:
I don't actually know what the behavior is when IPv6 is enabled but not configured anywhere. Maybe that `if` test is OK?

If `listen [::]:80` does not error when IPv6 is not available that sure looks like the cleanest option. I'll try and figure out a test.